Stratis.Bitcoin/Features/RPC/Models/PeerNodeModel.cs

```diff
@@ -0,0 +1,132 @@
+        /// The total number of bytes we’ve sent to this node.
+        /// Currently not populated.
+        /// </summary>
+        [JsonProperty(PropertyName = "bytessent")]
+        public long BytesSent { get; internal set; }
+
+        /// <summary>
+        /// The total number of bytes we’ve received from this node.
+        /// Currently not populated.
+        /// </summary>
+        [JsonProperty(PropertyName = "bytesrecv")]
+        public long BytesReceived { get; internal set; }
+        
+        /// <summary>
+        /// The connection time in seconds since epoch.
+        /// Currently not populated.
+        /// </summary>
+        [JsonProperty(PropertyName = "conntime")]
+        public int ConnectionTime { get; internal set; }
+
+        /// <summary>
+        /// The time offset in seconds.
+        /// </summary>
+        [JsonProperty(PropertyName = "timeoffset")]
+        public int TimeOffset { get; internal set; }
+
+        /// <summary>
+        /// The protocol version number used by this node.
+        /// </summary>
+        [JsonProperty(PropertyName = "version")]
+        public uint Version { get; internal set; }
+
+        /// <summary>
+        /// The user agent this node sends in its version message.
+        /// </summary>
+        [JsonProperty(PropertyName = "subver")]
+        public string SubVersion { get; internal set; }
+
+        /// <summary>
+        /// Whether node is inbound or outbound connection.
+        /// </summary>
+        [JsonProperty(PropertyName = "inbound")]
+        public bool Inbound { get; internal set; }
+
+        /// <summary>
+        /// The starting height (block) of the peer.
+        /// </summary>
+        [JsonProperty(PropertyName = "startingheight")]
+        public int StartingHeight { get; internal set; }
+            
+        /// <summary>
+        /// The last header we have in common with this peer.
+        /// Currently not populated.
+        /// </summary>
+        [JsonProperty(PropertyName = "synced_headers")]
+        public int SynchronizedHeaders { get; internal set; }
+
+        /// <summary>
+        /// The last block we have in common with this peer.
+        /// Currently not populated.
+        /// </summary>
+        [JsonProperty(PropertyName = "synced_blocks")]
+        public int SynchronizedBlocks { get; internal set; }
+
+        /// <summary>
+        /// Whether the peer is whitelisted.
+        /// </summary>
+        [JsonProperty(PropertyName = "whitelisted")]
+        public bool IsWhiteListed { get; internal set; }
+
+        /// <summary>
+        /// The heights of blocks we're currently asking from this peer.
+        /// Currently not populated.
+        /// </summary>
+        [JsonProperty(PropertyName = "inflight")]
+        public uint[] Inflight { get; internal set; }
+
+        /// <summary>
+        /// Number of blocks on peer.
+        /// </summary>
+        [JsonProperty(PropertyName = "blocks")]
```

Comment:
I can't see blocks in the cpp code 🤔 
Is the block height what is actually supposed to be in "inflight"?